PopPUNK/sketchlib.py

```diff
@@ -568,6 +583,41 @@ def queryDatabase(rNames, qNames, dbPrefix, queryPrefix, klist, self = True, num
         query_db = queryPrefix + "/" + os.path.basename(queryPrefix)
         distMat = pp_sketchlib.queryDatabase(ref_db, query_db, rNames, qNames, klist,
                                              True, False, threads, use_gpu, deviceid)
+                                             
+        # option to plot core/accessory fits. Choose a random number from cmd line option
+        if number_plot_fits > 0:
```

Comment:
Self and non-self queries, or am I missing something?

Reply:
Ah I see now. Still, I think it would make more sense to have this as a function and pass it different args for the self and non-self cases, as it's mostly the same code